packages/corvid-local-site/src/siteConverter.js

```diff
@@ -314,15 +323,17 @@ const editorCodeFilesToLocalCodeFiles = (
   editorCodeFiles,
   existingLocalPageFilePaths
 ) =>
-  editorCodeFiles.map(editorFile => ({
-    path: editorCodePathToLocalCodePath(
-      editorFile.path,
-      existingLocalPageFilePaths
-    ),
-    content: isUnderPath(".schemas", editorFile.path)
-      ? tryToPrettifyJsonString(editorFile.content)
-      : editorFile.content
-  }));
+  editorCodeFiles
+    .map(editorFile => ({
+      path: editorCodePathToLocalCodePath(
+        editorFile.path,
+        existingLocalPageFilePaths
+      ),
+      content: isUnderPath(".schemas", editorFile.path)
+        ? tryToPrettifyJsonString(editorFile.content)
+        : editorFile.content
+    }))
+    .concat(getCodeFilesTsConfigs());
```

Comment:
@nemni8 	why not making these part of the skeleton ?

Reply:
i didn't want to change the site skeleton as a dependent of using corvid types module, so i found this solution more "clean" from the siteConvertor perspective.